src/VisualStudio/Core/Def/Implementation/Watson/WatsonReporter.cs

```diff
@@ -114,6 +114,39 @@ public static void ReportNonFatal(Exception exception)
             session.PostEvent(faultEvent);
         }
 
+        private static string GetDescription(Exception exception)
```

Comment:
suggestion: rather than use the Description bucket for this, you could add the frame name to the end of the event, like "vs.ide.vbcs.nonfatalwatson" it could be "vs.ide.vbcs.nonfatalwatson.<framename>"
That has the additional advantages of easier Kusto querying and freeing up a param for other distinguishing info.
S

Reply:
whoops: typo: instead of "vs.ide.vbcs.nonfatalwatson" it could be "vs.ide.vbcs.FrameName"